python/ray/dashboard/modules/metrics/dashboards/serve_llm_dashboard_panels.py

```diff
@@ -274,6 +274,43 @@
         stack=False,
         grid_pos=GridPos(12, 40, 12, 8),
     ),
+    Panel(
+        id=28,
+        title="vLLM: Prefix Cache Hit Rate",
```

Comment:
we would like to show a percentile graph aggregating the data over replica_ids of the same deployments? 

For example if you have 10 replicas, you want to see average, max, min hit rate and not just absolute hit rate for all of them.